### PR TITLE
[Feature] CX-5282 Convert PhoneNumberInput component

### PR DESCRIPTION
**Pull request by josokinas**
Converting PhoneNumberInput component.

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1106-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1106-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1106-pr-onfido-sdk-ui-onfido.surge.sh

File: .stylelintrc [hunk 1]
```diff
@@ -2,6 +2,6 @@ extends: stylelint-config-sass-guidelines
 
 rules:
   max-nesting-depth: 3
-  selector-class-pattern: ^[a-z]+(?:[-|A-Z][a-z]+)*$
+  selector-class-pattern: ^[a-z]+(?:[A-Z|\-|\_]{1,2}[a-z]+)*$
```
Comment (kopijunkie) [inline .stylelintrc:5]: Ah, I reviewed the other PR before I saw this. 
But I guess that the latest regex pattern we want is the one from the Overlay conversion: `^[a-z0-9]+(?:[A-Z|\-|\_]{1,2}[a-z0-9]+)*$` when this gets merged in?

Comment (josokinas) [reply .stylelintrc:5]: Yep, I will resolve to later on. There were so many different cases, e.g. overriding some 3rd party dependency styles... But still did not want to let just _any_ class pattern.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1106-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1106-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1106-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**